test/testsuites/invalidopsrctests.go

```diff
@@ -78,9 +77,7 @@ func testOpSrcWithInvalidEndpoint(t *testing.T) {
 		}
 		return false, nil
 	})
-	if err != nil {
-		t.Errorf("OperatorSource never reached expected phase/message, expected %v: %v", expectedPhase, err)
-	}
+	assert.NoError(t, err, fmt.Sprintf("OperatorSource never reached expected phase/message, expected %v: %v", expectedPhase, err))
```

Comment:
This will never print the actual phase. Applies in all instances. I do not believe that it needs to be fixed in this PR.

Reply:
@awgreene Not sure I follow. Why won't it print `OperatorSource never reached expected phase/message, expected Downloading: ....error message...` ?

Reply:
+1. You can also drop the `err` as you have done elsewhere. Applies to all instances.

Reply:
@awgreene @aravindhp dropped `err` from all similar instances.